src/features/island/hud/components/deliveries/TravelModal.tsx

```diff
@@ -29,6 +29,7 @@ export const TravelModal: React.FC<Props> = ({ isOpen, onClose }) => {
         centered
         show={isOpen}
         onHide={onClose}
+        OnShow={() => gameService.send("SAVE")}
```

Comment:
It was a capitalisation issue
"OnShow" change to "onShow"

Reply:
Weird.  I quadruple checked that.  Fixing now.